src/goto-instrument/model_argc_argv.cpp

```diff
@@ -0,0 +1,157 @@
+/*******************************************************************\
```

Comment:
Isn't this C-specific and should therefore go into ansi-c/ ?
A language-independent version would be appreciate so that it works also for Java.